packages/ui/CONTRIBUTING.md

```diff
@@ -5,3 +5,24 @@
 1. Navigate to the _root_ of your local clone of [aws-amplify/amplify-ui](https://github.com/aws-amplify/amplify-ui)
 1. Run `yarn setup`
 1. Run `yarn ui dev`
+
+## Style Guide
+
+The @aws-amplify/ui package uses the [BEM](http://getbem.com/) styling syntax to standardize its css code and maintain flat css selectors.  The goal is to keep all css selectors at a 010 specificity so that customers can easily restyle their application using any of the available styling tools that exist. This is achieved by using specific classes that follow the syntax of `{Block}__{Element}--{Modifier}`.  Blocks should be a standalone entity that has meaning on its own. An Element is a part of a block and only has meaning within the context of a block.  A Modifier is state on a block or element that changes its behavior or appearance.
```

Comment:
```suggestion
The @aws-amplify/ui package uses the [BEM](http://getbem.com/) styling syntax to standardize its CSS code and maintain flat CSS selectors.  The goal is to keep all CSS selectors at a 010 specificity so that customers can easily restyle their application using any of the available styling tools that exist. This is achieved by using specific classes that follow the syntax of `{Block}__{Element}--{Modifier}`.  Blocks should be a standalone entity that has meaning on its own. An Element is a part of a block and only has meaning within the context of a block.  A Modifier is state on a block or element that changes its behavior or appearance.
```